src/main/scala/edu/berkeley/ce/rockslicing/LoadBalancer.scala

```diff
@@ -0,0 +1,323 @@
+package edu.berkeley.ce.rockslicing
+
+import breeze.linalg
+import breeze.linalg.{DenseVector, DenseMatrix}
+import scala.annotation.tailrec
+
+/** 
+  * Manages initial partitioning of rock volume to maintain load balance among partitions
+  */
+object LoadBalancer {
+
+  /**
+    * Produce processor joints that cut initial rock volume into approximately equal volumes.
+    * These joints will be used to seed the initial Block RDD such that the volume or rock
+    * provided to each process is approximately equal.
+    * @param rockVolume Initial block defining the entire rock volume of interest
+    * @param numSeedJoints Number of joints to process before initiating parallel block cutting.
+    *                      Should be one less than number of cores used.
+    * @return Set of processor joints that will be used to seed the initial RDD
+    */
+  def generateSeedJoints(rockVolume: Block, numSeedJoints: Integer):
```

Comment:
Hmm, I think we should try to be consistent on our terminology of seed joints vs processor joints.

In fact, do we even need our old notion of seed joints now? We could just use processor joints to divide up the work automatically, right? That would definitely simplify things from the user's end.